src/main/java/mate/jdbc/model/Driver.java

```diff
@@ -46,22 +74,24 @@ public boolean equals(Object o) {
             return false;
         }
         Driver driver = (Driver) o;
-        return Objects.equals(id, driver.id)
-                && Objects.equals(name, driver.name)
-                && Objects.equals(licenseNumber, driver.licenseNumber);
+        return Objects.equals(id, driver.id) && Objects.equals(name, driver.name)
+                && Objects.equals(licenseNumber, driver.licenseNumber)
+                && Objects.equals(car, driver.car);
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(id, name, licenseNumber);
+        return Objects.hash(id, name, licenseNumber, car);
```

Comment:
Remove the car attribute from the hashCode() method, as the relationship between Driver and Car should not be in the Driver class.